Polish resources repository.md

```diff
@@ -0,0 +1,34 @@
+## Icons Legend
```

Comment:
What if we name this file "Readme.pl.md" or "Readme_pl.md" or something like that? Because this is, in fact, Readme in different language.

Reply:
Good idea!